### DISCUSSION TITLE
Binding Problem Ikea Shortcut

### DISCUSSION THREAD

**Discussion by onkel-joerg** [Koenkk/zigbee2mqtt#8564] [General]
Hello,
I have a problem with a TRADFRI SHORTCUT Button model 1812.
I was able to bind a shortcut button to a innr plug sp120.
One click on the button switches the plug on but no matter what I do, it doesn't switch it off!

What am I doing wrong?

[Screenshot_20210829-172520_Chrome.jpg](https://user-images.githubusercontent.com/78688786/131256159-187ad3b6-5e64-462a-8207-d671f21991d8.jpg)

**Comment by sjorge** [comment]
Nothing, the shortcut button only sends genOnOff on. You need one of the on/off buttons if you want to turn things on and off.

**Reply by michal-bartak** [reply]
or possibly you can send it to some automation system to trigger toggle on desired device

**Reply by sjorge** [reply]
That’s a possibility yeah, but then you’re not using direct bindings between the devices.

**Reply by onkel-joerg** [reply]
> Nothing, the shortcut button only sends genOnOff on. You need one of the on/off buttons if you want to turn things on and off.

I thought I can change this somehow to a "toggle event" because the Mūller Licht Remote switches the binded Osram Bulb on and off wi t h the same key.

Thanks

**Reply by sjorge** [reply]
> > Nothing, the shortcut button only sends genOnOff on. You need one of the on/off buttons if you want to turn things on and off.
> 
> I thought I can change this somehow to a "toggle event" because the Mūller Licht Remote switches the binded Osram Bulb on and off wi t h the same key.
> 

That would be nice if we could, but sadly as far as I've looked at the ZCL and the clusters these remotes have, there is no way to do so.
